imports/plugins/included/default-theme/client/styles/cards.less

```diff
@@ -83,7 +83,7 @@
 .rui.card .content {
   .flex(1 1 auto);
   padding: 10px;
-  cursor: pointer;
+  cursor: default;
```

Comment:
`That the cursor: default is not necessary if its not overriding anything.` but it is.. because the cards are clickable by default.. the logic to add/remove the anchor is missing from the card component.